joomla/joomla-cms
# [6.0] Numbers custom fields plugin

I'm trying to create a new custom fields plugin for the standard Joomla Form Field Number. We have it for Integer and i'd like to have the same for the number field. That way w'll also gain ability to use floats. 
 
### Summary of Changes
Extra plugin added for the numbers standard form field type. With added currency options for display. Making it very useful for display of menu's for restaurants or anything else where a currency or a number is needed to be displayed.

### Testing Instructions
You can discover the plugin and then use it in custom fields. It's called numbers. 
Create a custom field of type numbers and check weather the options available do what they supposed to be doing. especially related to the number formatting.

### Actual result BEFORE applying this Pull Request
No numbers custom field available.

### Expected result AFTER applying this Pull Request
Numbers custom field available and working.


### Link to documentations
Please select:
